Validation/RecoParticleFlow/python/miniAODDQM_cff.py

```diff
@@ -3,90 +3,86 @@
+    CreatePFractionHistos = True
+)
 
 
 # jetRes plots
 from DQMOffline.PFTau.PFJetResDQMAnalyzer_cfi import pfJetResDQMAnalyzer
 
-JetResValidation1 = pfJetResDQMAnalyzer.clone()
-JetResValidation1.InputCollection = JetValidation1.InputCollection
-JetResValidation1.MatchCollection = JetValidation1.MatchCollection
-JetResValidation1.ptMin = JetValidation1.ptMin
+slimmedJetResValidation1 = pfJetResDQMAnalyzer.clone(
+    InputCollection = slimmedJetValidation1.InputCollection,
+    MatchCollection = slimmedJetValidation1.MatchCollection,
+    ptMin = slimmedJetValidation1.ptMin
+)
 
-JetResValidation2 = pfJetResDQMAnalyzer.clone()
-JetResValidation2.InputCollection = JetValidation2.InputCollection
-JetResValidation2.MatchCollection = JetValidation2.MatchCollection
-JetResValidation2.ptMin = JetValidation2.ptMin
+slimmedJetResValidation2 = pfJetResDQMAnalyzer.clone(
+    InputCollection = slimmedJetValidation2.InputCollection,
+    MatchCollection = slimmedJetValidation2.MatchCollection,
+    ptMin = slimmedJetValidation2.ptMin
+)
 
 
 # MET
 from DQMOffline.PFTau.PFMETDQMAnalyzer_cfi import pfMETDQMAnalyzer
 
-METValidation1 = pfMETDQMAnalyzer.clone()
-METValidation1.BenchmarkLabel  = cms.string('slimmedMETValidation/CompWithPFMET')
-METValidation1.InputCollection = cms.InputTag('slimmedMETs')
-METValidation1.MatchCollection = cms.InputTag('pfMet')
+slimmedMETValidation1 = pfMETDQMAnalyzer.clone(
+    BenchmarkLabel  = 'slimmedMETValidation/CompWithPFMET',
+    InputCollection = 'slimmedMETs',
+    MatchCollection = 'pfMet'
+)
 
-METValidation2 = pfMETDQMAnalyzer.clone()
-METValidation2.BenchmarkLabel  = cms.string('slimmedMETValidation/CompWithPFMETT1')
-METValidation2.InputCollection = cms.InputTag('slimmedMETs')
-METValidation2.MatchCollection = cms.InputTag('pfMetT1')
+slimmedMETValidation2 = pfMETDQMAnalyzer.clone(
+    BenchmarkLabel  = 'slimmedMETValidation/CompWithPFMETT1',
+    InputCollection = 'slimmedMETs',
+    MatchCollection = 'pfMetT1'
+)
 
 
 # muons
 from DQMOffline.PFTau.PFMuonDQMAnalyzer_cfi import pfMuonDQMAnalyzer
 
-slimmedMuonValidation1 = pfMuonDQMAnalyzer.clone()
-slimmedMuonValidation1.BenchmarkLabel  = cms.string('SlimmedMuonValidation/CompWithRecoMuons')
-slimmedMuonValidation1.InputCollection = cms.InputTag('slimmedMuons')
-slimmedMuonValidation1.MatchCollection = cms.InputTag('muons')
-# official
-#muonPFsequenceMC.inputTagMuonReco = cms.InputTag('slimmedMuons')
-#muonPFsequenceMC.inputTagGenParticles = cms.InputTag('muons')
-#muonPFsequenceMC.runOnMC = cms.bool(False)
-# RefCore: A request to resolve a reference to a product of type 'std::vector<reco::Track>' with ProductID '3:1469' can not be satisfied because the product cannot be found.
-# with the following:
-#muonPFsequenceMC.inputTagMuonReco = cms.InputTag('muons')
-#muonPFsequenceMC.inputTagGenParticles = cms.InputTag('slimmedMuons')
+slimmedMuonValidation1 = pfMuonDQMAnalyzer.clone(
+    BenchmarkLabel  = 'slimmedMuonValidation/CompWithRecoMuons',
```

Comment:
@hatakeyamak  just to confirm: Slimmed -> slimmed is intentional and not a typo, right?

Reply:
It's intentional, as all other 'slimmed` folder started with a lower case.